ncm-ccm/src/main/pan/components/ccm/schema.pan

```diff
@@ -35,7 +30,7 @@ type kerberos_principal_string = string with {
         error("Not a valid principal " + principal);
     };
 
-    realm = pc_r[len-1];
+    realm = pc_r[len - 1];
```

Comment:
hmm.. that looks a bit odd. Normally languages (e.g. pep8) allow no spaces in this context don't they?